doc/deps.md

```diff
@@ -81,6 +95,20 @@ sudo apt-get install libodb-sqlite-dev libsqlite3-dev
 sudo apt-get install libodb-pgsql-dev postgresql-server-dev-<version>
 ```
 
+##### Ubuntu 18.04 ("Bionic Beaver") and 20.04 ("Focal Fossa") LTS
+
+The database connector library must be compiled manually on these systems,
+however, the database programs themselves should be installed from the
+package manager.
```

Comment:
@bruntib  confirmed that ODB 2.4 works under Ubuntu 20.04, so it's fine to manually compile ODB only on 18.04.